### ISSUE TITLE
Rich UI

### ISSUE THREAD

**Issue by sinasalek** [borgbackup/borgweb#99]
Duplicati version 2, comes with an excellent webbased UI with allows fully managing backups.
I think it's a very good sample, that would have been great if borg also had such a UI
It might even be possible adopt its UI code for borg
https://github.com/duplicati/duplicati
http://www.duplicati.com/news/duplicati20experimentalreleasewithui

**Comment by ThomasWaldmann** [comment]
There's a lot else to do first, even if we considered extending the scope to more than "view the logs, start a backup".

So, that needs to be done first (see milestones) and then there would be a need for many more developers - doing a more extensive UI is also a lot of work.

**Comment by ThomasWaldmann** [comment]
duplicate of #91.
